Add reasons to qualify an ad as "illegal" (#38)
ambanum/disinformation-encyclopedia: _pages/encyclopedia/3-qualification/3-tools.md

In the future, EDNH will open to other media outlets and will allow users (citizens) to answer surveys on salient European issues, which will then be translated into readable data.

EDNH is independent and funded by the European Commission.

## Criterion: Illegal

### In France

The various criteria detailed below allow for the qualification of a political ad or an issue ad as illegal. However they are **only valid in France**, as they rely **exclusively** on French jurisdiction. Therefore, the following content is in French.

#### Publicité politique en période électorale

##### Description
{: .no_toc}

Il est interdit d'utiliser des procédés de publicité commerciale à des fins de **propagande électorale** en période électorale, c'est-à-dire la période allant des « six mois précédant le premier jour du mois d'une élection et jusqu'à la date du tour de scrutin où celle-ci est acquise » .

[Article L52-1 du Code électoral](https://www.legifrance.gouv.fr/affichCodeArticle.do?cidTexte=LEGITEXT000006070239&idArticle=LEGIARTI000006353102&dateTexte=&categorieLien=cid), tel que modifié par la loi n°2011-412 du 14 avril 2011 - art. 6.

##### Peines encourues
{: .no_toc}

> Depuis 2017, tout candidat (ou tête de liste) ayant demandé ou donné son accord pour de la publicité commerciale ou de l'affichage en période électorale encourt un an d'emprisonnement et 15 000 € d'amende.

Author: Certaines élections se font par listes, d'autres non.

Une liste comporte des candidats, mais la loi ne concerne que les candidats en tête de liste. 

S'il n'y a pas de liste (donc pas de tête de liste), alors tous les candidats sont concernés.

D'où la formulation: "tout candidat ou tête de liste".

- [Article L. 113-1.-II du Code électoral](https://www.legifrance.gouv.fr/affichCodeArticle.do?cidTexte=LEGITEXT000006070239&idArticle=LEGIARTI000027477771), tel que modifié par l'article 25 de la loi n°2017-1339 du 15 septembre 2017 pour la confiance dans la vie politique. 

#### Utilisation des outils publicitaires des plateformes en période électorale

##### Description
{: .no_toc}

Durant la période électorale, l'interdiction de l'utilisation de publicité commerciale à des fins de propagande électorale comprend la diffusion de ce type de publicité par **tout moyen de communication audiovisuelle**. Cela implique l'interdiction d’utiliser les outils publicitaires des plateformes à cette période pour diffuser de la propagande électorale. 

> De plus, « les interdictions et restrictions prévues [par le code de l'électorat] en matière de propagande électorale sont applicables à tout message ayant le caractère de propagande électorale diffusé par **tout moyen de communication au public par voie électronique**.» 

Author: Il s'agit du même article de loi, mais:
- le premier point appuie l'**interdiction de pub politique** en période électorale (large)
- le deuxième point appuie l'interdiction du recours à des **moyens de communication audiovisuelle et/ou électronique** pour diffuser des pub politiques (plus précis)

> De plus, « les interdictions et restrictions prévues [par le code de l'électorat] en matière de propagande électorale sont applicables à tout message ayant le caractère de propagande électorale diffusé par **tout moyen de communication au public par voie électronique**.» 

Author: Ce sont donc bien 2 critères distincts. Après comme pour nous le plus intéressant est le deuxième, je peux supprimer le premier

In the future, EDNH will open to other media outlets and will allow users (citizens) to answer surveys on salient European issues, which will then be translated into readable data.

EDNH is independent and funded by the European Commission.

## Criterion: Illegal

> ### 1 - Publicité politique en période électorale

Reviewer: Precise that the following content is in french `[Français]`

Reviewer 2: Even more precisely: state that this is valid only for France. Right now, this reads as general validity while it is valid only within the French jurisdiction.
Suggested implementation: add a level 3 title “In France” with an introductory paragraph explaining that the following content will be in French.

> ### 1 - Publicité politique en période électorale

Reviewer 2: Avoid numbered titles. They don't help in readability as heading levels already make the hierarchy clear, and anchors allow for direct reference without a need for numbers. Conversely, adding or removing a paragraph then needs updating all numbers.

> #### Description

Reviewer: Add `{: .no_toc}` under each title level 4 like:
```
#### Description
{: .no_toc}
```

> Il est interdit d'utiliser "**à des fins de propagande électorale [...] tout procédé de publicité commerciale** par la voie de la presse ou par tout moyen de communication audiovisuelle" en période électorale, c'est-à-dire la période allant des "six mois précédant le premier jour du mois d'une élection et jusqu'à la date du tour de scrutin où celle-ci est acquise".

Reviewer 2: If in a French text, use French quotes: «», not "".

> Il est interdit d'utiliser "**à des fins de propagande électorale [...] tout procédé de publicité commerciale** par la voie de la presse ou par tout moyen de communication audiovisuelle" en période électorale, c'est-à-dire la période allant des "six mois précédant le premier jour du mois d'une élection et jusqu'à la date du tour de scrutin où celle-ci est acquise".

Reviewer 2: Use ellipsis: `…` instead of three dots (`...`).

> > _Référence de l’article de loi_

Reviewer: Remove each `> _Référence de l’article de loi_`

Reviewer 2: Not only is it not very readable, it is counter-semantic: this is not a quotation, we are not _quoting_ content. Use a title at the same level as Description, since this is not a description anymore.

[Article L52-1 du Code électoral](https://www.legifrance.gouv.fr/affichCodeArticle.do?cidTexte=LEGITEXT000006070239&idArticle=LEGIARTI000006353102&dateTexte=&categorieLien=cid), tel que modifié par la loi n°2011-412 du 14 avril 2011 - art. 6.

#### Peines encourues

> Depuis 2016, toute infraction à l'article L52-1 exposait à une amende de 75 000 euros. 

Reviewer 2: I don't see the added value of the historical evolution; on the other hand, listing it makes it longer for the reader.

Depuis 2017, tout candidat (ou tête de liste) ayant demandé ou donné son accord pour de la publicité commerciale ou de l'affichage en période électorale encourt un an d'emprisonnement et 15 000 € d'amende.

> _Référence de l’article de loi_

1) [Article L90-1 du Code électoral](https://www.legifrance.gouv.fr/affichCodeArticle.do?cidTexte=LEGITEXT000006070239&idArticle=LEGIARTI000006353212&dateTexte=&categorieLien=cid) tel que modifié par la loi n°2016-508 du 25 avril 2016 - art. 4. 

> 2) [Article L. 113-1.-II du Code électoral](https://www.legifrance.gouv.fr/affichCodeArticle.do?cidTexte=LEGITEXT000006070239&idArticle=LEGIARTI000027477771), tel que modifié par l'article 25 de la loi n°2017-1339 du 15 septembre 2017 pour la confiance dans la vie politique. 

Reviewer 2: This syntax creates a paragraph with a number that will be too easily copied and make it harder to reference the article. Use an unordered list instead:

```
- Article L90-1 du Code électoral
- Article L. 113-1.-II du Code électoral
```

### 1 bis - Utilisation des outils publicitaires des plateformes en période électorale

#### Description

> Il est interdit d'utiliser "à des fins de propagande électorale [...] tout procédé de publicité commerciale par la voie de la presse ou par **tout moyen de communication audiovisuelle**" en période électorale (cf 3). Cela implique l'interdiction d’utiliser les outils publicitaires des plateformes à cette période pour diffuser de la propagande électorale. 

Reviewer 2: What's the difference with the above? 🤔 Merging might be more appropriate.

>_Référence de l’article de loi_

[Article L52-1 du Code électoral](https://www.legifrance.gouv.fr/affichCodeArticle.do?cidTexte=LEGITEXT000006070239&idArticle=LEGIARTI000006353102&dateTexte=&categorieLien=cid), tel que modifié par la loi n°2011-412 du 14 avril 2011 - art. 6.

#### Peines encourues

Comme explicité à la section 3, tout candidat (ou tête de liste) ayant volontairement et sciemment diffusé des publicités politiques via les réseaux sociaux en période électorale encourt un an d'emprisonnement et 15 000 € d'amende.

> _Référence de l’article de loi_

[Article L. 113-1.-II du Code électoral](https://www.legifrance.gouv.fr/affichCodeArticle.do?cidTexte=LEGITEXT000006070239&idArticle=LEGIARTI000027477771), tel que modifié par l'article 25 de la loi n°2017-1339 du 15 septembre 2017 pour la confiance dans la vie politique.  

**Nota Bene:**

> "Les interdictions et restrictions prévues [par le code de l'électorat] en matière de propagande électorale sont applicables à tout message ayant le caractère de propagande électorale diffusé par **tout moyen de communication au public par voie électronique**."

Reviewer 2: What is the point of adding this in an NB rather than in the description? Merge all that together.

> _Référence de l’article de loi_

[Article L48-1.-V- 2 du Code électoral](https://www.legifrance.gouv.fr/affichCodeArticle.do?cidTexte=LEGITEXT000006070239&idArticle=LEGIARTI000023878437).

### 2 - Violation de la période de silence

#### Description

> Une période de silence est observée à partir de la veille du scrutin à minuit (c'est-à-dire durant le samedi précédant les élections, qui ont lieu le dimanche). Toute communication politique est interdite durant cette période, y compris des commentaires, des sondages ou tout autre document. La diffusion au public par voie électronique de communication politique caractérisée comme de la propagande électorale entre dans ce cadre.

Reviewer 2: Les élections ont lieu le samedi pour une partie du territoire national (Amériques et Caraïbes). Préciser comment la période de silence s'applique sur ces territoires.

> Une période de silence est observée à partir de la veille du scrutin à minuit (c'est-à-dire durant le samedi précédant les élections, qui ont lieu le dimanche). Toute communication politique est interdite durant cette période, y compris des commentaires, des sondages ou tout autre document. La diffusion au public par voie électronique de communication politique caractérisée comme de la propagande électorale entre dans ce cadre.

Reviewer 2: Il faut expliciter ce qu'est la « propagande électorale », alors !

> _Référence de l’article de loi_

[Article L49 - 4 du Code électoral](https://www.legifrance.gouv.fr/affichCode.do;jsessionid=79E5ACC204BC0F2F68D4ABFE97D36A49.tplgfr41s_1?idSectionTA=LEGISCTA000006148458&cidTexte=LEGITEXT000006070239&dateTexte=20190514).

#### Peines encourues

> Toute personne enfreignant les dispositions de l'article L. 49 encourt une amende de 3 750 euros.

Reviewer 2: > No need for “Toute personne enfreignant les dispositions”  in a paragraph titled “Peines encourues”.

> _Référence de l’article de loi_

[Article L89](https://www.legifrance.gouv.fr/affichCode.do;jsessionid=E9453AC1030D9ACC4E1AC088A6B5526A.tplgfr41s_1?idSectionTA=LEGISCTA000006148461&cidTexte=LEGITEXT000006070239&dateTexte=20190514) du Code électoral, tel que modifié par l'article 4 de la loi n°2016-508 du 25 avril 2016.

> #### 3 - Introduction d'un élément polémique nouveau

Reviewer: Wrong title level `####3` --> `###3`

#### Description

> Il est interdit aux candidats de révéler "un élément nouveau de polémique électorale" à un moment trop proche de la fin de la campagne électorale, si cela implique que le(s) candidat(s) touché(s) ne pourront pas y répondre utilement.

Reviewer 2: For readability, use plural form only.
```suggestion
Il est interdit aux candidats de révéler « un élément nouveau de polémique électorale » à un moment trop proche de la fin de la campagne électorale, si cela implique que les candidat‧e‧s touché‧e‧s ne pourront pas y répondre utilement.
```

> Il est interdit aux candidats de révéler "un élément nouveau de polémique électorale" à un moment trop proche de la fin de la campagne électorale, si cela implique que le(s) candidat(s) touché(s) ne pourront pas y répondre utilement.

Reviewer 2: Que signifie « utilement » ?

Author: Bonne question, c'est le texte de loi qui emploie ce mot, je pense que c'est volontairement flou. Je mets des guillemets

> _Référence de l’article de loi_
[Article L48-2 - art. 3 du Code électoral.](https://www.legifrance.gouv.fr/affichCode.do;jsessionid=79E5ACC204BC0F2F68D4ABFE97D36A49.tplgfr41s_1?idSectionTA=LEGISCTA000006148458&cidTexte=LEGITEXT000006070239&dateTexte=20190514).

### 4 - Sondages non conformes

#### Description

> En dehors de la période électorale, il est interdit de publier des sondages d'opinion si leur objectivité, leur qualité et leur authenticité n'a pas été garantie par la Commission des Sondages. En effet, l'entité publiant ou diffusant les résultats d'un sondage doit avoir soumis une déclaration à la Commission, et que celle-ci y ait souscrit.

Reviewer 2: Mais pendant la période électorale c'est autorisé ? 🤔

> En dehors de la période électorale, il est interdit de publier des sondages d'opinion si leur objectivité, leur qualité et leur authenticité n'a pas été garantie par la Commission des Sondages. En effet, l'entité publiant ou diffusant les résultats d'un sondage doit avoir soumis une déclaration à la Commission, et que celle-ci y ait souscrit.

Reviewer 2: Comment puis-je m'assurer qu'un sondage respecte cela ? Mettre un lien vers la commission en question, et vers la liste des déclarations souscrites.

Author: Il n'y a pas de liste des déclarations mais j'ai mis le lien vers le site de la Commission, où les conditions sont re-précisées

[Article 7 de la loi n° 77-808](https://www.legifrance.gouv.fr/affichTexte.do?cidTexte=JORFTEXT000000522846) du 19 juillet 1977 relative à la publication et à la diffusion de certains sondages d'opinion, telle que modifié par l'article 24 de la loi n°2017-55 du 20 janvier 2017.

#### Peines encourues

La réalisation, la publication ou la diffusion d'un sondage non-conforme est punie d'une amende de 75 000€.

> _Référence de l’article de loi_

[Article 12 de la loi n° 77-808](https://www.legifrance.gouv.fr/affichTexte.do?cidTexte=JORFTEXT000000522846) du 19 juillet 1977 relative à la publication et à la diffusion de certains sondages d'opinion, telle que modifiée par l'article 6 de la loi n°2016-508 du 25 avril 2016.

### 5 - Publicité politique au financement étranger

#### Description

En France, les dons faits à un parti politique ou à une campagne et provenant d'un pays étranger ou de personnes morales basées à l'étranger sont interdits. 

> Une personne physique ne peut verser de don ou de cotisation à un parti ou groupement politique que si elle est de nationalité française ou si elle réside en France.

Reviewer 2: Simplifier la formulation en fusionnant les deux paragraphes.

Author: Mais les articles n'ont pas été remaniés par les mêmes lois...

> _Référence de l’article de loi_

[Article 11-4 de la loi n° 88-227](https://www.legifrance.gouv.fr/affichTexte.do?cidTexte=JORFTEXT000000321646#LEGIARTI000035588686) du 11 mars 1988 relative à la transparence financière de la vie politique, tel que modifié par l'article 25 de la loi n°2017-1339 du 15 septembre 2017.

#### Peines encourues

Les personnes qui ont versé un don ou consenti un prêt à un ou plusieurs partis ou groupements politiques en violation des articles 11-3-1 et 11-4 sont punies de trois ans d'emprisonnement et de 45 000 € d'amende.

> Les même peines sont applicables au bénéficiaire du don ou du prêt consenti par une personne physique ou morale, y compris de droit étranger.

Reviewer 2: Simplifier la formulation en fusionnant les deux paragraphes : « Les bénéficiaires et les donateurs risquent trois ans de prison et 45 000 € d'amende. »

> _Référence de l’article de loi_

[Article 11-5 de la loi n° 88-227](https://www.legifrance.gouv.fr/affichTexte.do?cidTexte=JORFTEXT000000321646#LEGIARTI000035588686) du 11 mars 1988 relative à la transparence financière de la vie politique, tel que modifié par l'article 25 de la loi n°2017-1339 du 15 septembre 2017 pour la confiance dans la vie politique.

### 6 - Publicité politique financée par une personne morale 

#### Description

> Les personnes morales, même françaises, à l'exception des partis ou groupements politiques ne peuvent pas contribuer au financement des partis ou groupements politiques ou leur faire des dons **sous quelque forme que ce soit**. 

Reviewer 2: Simplifier la formulation en évitant la double négation.

> Les personnes morales, même françaises, à l'exception des partis ou groupements politiques ne peuvent pas contribuer au financement des partis ou groupements politiques ou leur faire des dons **sous quelque forme que ce soit**. 

Reviewer 2: Le gras ne semble pas particulièrement utile ici, et crée une différence de poids par rapport aux autres entrées.

> _Référence de l’article de loi_

[Article 11-4 de la loi n° 88-227](https://www.legifrance.gouv.fr/affichTexte.do?cidTexte=JORFTEXT000000321646#LEGIARTI000035588686) du 11 mars 1988 relative à la transparence financière de la vie politique, tel que modifié par l'article 25 de la loi n°2017-1339 du 15 septembre 2017 pour la confiance dans la vie politique.

#### Peines encourues

Les personnes qui ont versé un don ou consenti un prêt à un ou plusieurs partis ou groupements politiques en violation des articles 11-3-1 et 11-4 sont punies de trois ans d'emprisonnement et de 45 000 € d'amende.

> Les même peines sont applicables au bénéficiaire du don ou du prêt consenti par une personne physique ou morale, y compris de droit étranger.

Reviewer 2: Simplifier la formulation en fusionnant les deux paragraphes.

> _Référence de l’article de loi_

[Article 11-5 de la loi n° 88-227](https://www.legifrance.gouv.fr/affichTexte.do?cidTexte=JORFTEXT000000321646#LEGIARTI000035588686) du 11 mars 1988 relative à la transparence financière de la vie politique, tel que modifié par l'article 25 de la loi n°2017-1339 du 15 septembre 2017 pour la confiance dans la vie politique.

### 7 - Publicité politique fausse ou trompeuse

#### Description

> L'interdiction des pratiques commerciales déloyales, parmi lesquelles sont comprises les pratiques commerciales trompeuses, inclut l'interdiction des publicités fausses ou trompeuses. 

Reviewer 2: Quelle est la référence qui garantit l'interprétation du Code de la consommation dans le contexte des publicités politiques ? 🤔

Author: Dans le texte de loi précédent on mentionnait les publicités exclusivement, avec la loi Hamon  puis l'ordonnance on a donné un terme plus large

Reviewer 2: Je suis très surpris que les publicités politiques tombent le coup du code de la consommation. As-tu une référence sur cette « loi Hamon » et « l'ordonnance » qui permettrait d'éclairer cette interprétation ?

Comment peut-on qualifier un programme politique de « faux ou trompeur » ? Il ne s'agit pas d'une « pratique commerciale » !

Author: Auparavant dans le **Code de la Consommation**, [article L-121-1](https://www.legifrance.gouv.fr/affichCode.do?idSectionTA=LEGISCTA000006161818&cidTexte=LEGITEXT000006069565&dateTexte=20080104)

"Est interdite toute publicité comportant, sous quelque forme que ce soit, des allégations, indications ou présentations fausses ou de nature à induire en erreur" (Version en vigueur en janvier 2008)

[**_Loi Hamon_**](https://www.legifrance.gouv.fr/affichTexte.do?cidTexte=JORFTEXT000028738036&categorieLien=id) portant modification du **Code de la Consommation**, en 2014:

" le mot : « publicité » est remplacé par les mots : « communication à caractère promotionnel » ", sens plus large

[**Ordonnance n° 2016-301 du 14 mars 2016** relative à la partie législative du **code de la consommation**], portant aussi modification du **Code de la Consommation**(https://www.legifrance.gouv.fr/affichTexte.do;jsessionid=0F56FE1192540941799FB5B3DD103072.tplgfr41s_3?cidTexte=JORFTEXT000032209352&dateTexte=20160316)

Les pratiques commerciales déloyales sont interdites. (article L-121-1)
Critères d'une pratique commerciale trompeuse (article L-121-2)

[Art. L. 113-1.-I.-](https://www.legifrance.gouv.fr/affichTexte.do?cidTexte=JORFTEXT000035567974&categorieLien=id)  de la **_LOI n° 2017-1339 du 15 septembre 2017 pour la confiance dans la vie politique_** :

Sera puni de trois ans d'emprisonnement et de 45 000 € d'amende tout candidat, en cas de scrutin uninominal ou binominal, ou tout candidat tête de liste, en cas de scrutin de liste, qui [...] Aura bénéficié, sur sa demande ou avec son accord exprès, d'affichages ou de **publicité commerciale** ne respectant pas les articles L. 51 et L. 52-1 ()

Article 52-1: (https://www.legifrance.gouv.fr/affichCodeArticle.do?cidTexte=LEGITEXT000006070239&idArticle=LEGIARTI000006353102&dateTexte=&categorieLien=cid) Interdit "l'utilisation à des fins de propagande électorale de tout procédé de publicité commerciale"

(Et par ailleurs: " la publicité commerciale qui utilise la politique pour promouvoir des produits et services : un annonceur peut-il faire appel à l’actualité politique, à un débat politique ou à des personnages politiques dans sa communication ?... tout propos politique est strictement interdit dans les espaces publicitaires par la loi dans les médias audiovisuels pour des raisons historiques liées à l’origine publique de la télévision" [Code ARPP](https://www.arpp.org/wp-content/uploads/2017/12/Code-ARPP.pdf) )

In the future, EDNH will open to other media outlets and will allow users (citizens) to answer surveys on salient European issues, which will then be translated into readable data.

EDNH is independent and funded by the European Commission.

## Criterion: Illegal

### In France

> The various criteria detailed below allow for the qualification of a political ad or an issue ad as illegal. However they are only valid in France, as they rely exclusively on French jurisdiction. Therefore, the following content is in French.

Reviewer 2: ```suggestion
The various criteria detailed below allow for the qualification of a political ad or an issue ad as illegal. However they are only valid within French jurisdiction. Therefore, the following content is in French.
```

#### Publicité politique en période électorale
{: .no_toc}

Il est interdit d'utiliser des procédés de publicité commerciale à des fins de propagande électorale en période électorale, _i.e_ jusqu'au jour de l’élection et dans les six mois pleins qui la précèdent.

> La diffusion de ce type de publicité à cette période par tout moyen de communication audiovisuelle ou électronique, y compris les outils publicitaires des plateformes, est donc interdite.

Reviewer 2: Pas nécessaire, ou éventuellement à réunir avec le paragraphe précédent.

Par exemple, si le premier tour a lieu le 28 août et le second tour le 2 septembre, toute publicité à but électoral est interdite du 1<sup>er</sup> février au 2 septembre, inclus.

> [Article L52-1 du Code électoral](https://www.legifrance.gouv.fr/affichCodeArticle.do?cidTexte=LEGITEXT000006070239&idArticle=LEGIARTI000006353102&dateTexte=&categorieLien=cid), tel que modifié par la loi n°2011-412 du 14 avril 2011 - art. 6.

Reviewer 2: Formatage : utiliser une liste comme au-dessous.

- [Article L48-1.-V- 2 du Code électoral](https://www.legifrance.gouv.fr/affichCodeArticle.do?cidTexte=LEGITEXT000006070239&idArticle=LEGIARTI000023878437).

##### Peines encourues
{: .no_toc}

> Tout candidat ayant demandé ou donné son accord pour de la publicité commerciale ou de l'affichage en période électorale encourt un an d'emprisonnement et 15 000 € d'amende. Si une liste électorale a été constituée, seul le candidat en tête de liste est concerné.

Reviewer 2: Neutralité de genre.
